frontend/app/src/components/defi/DefiModuleSelector.vue

```diff
@@ -0,0 +1,62 @@
+<script setup lang="ts">
+import { useListeners } from 'vue';
+import {
+  type Module,
+  SUPPORTED_MODULES,
+  type SupportedModule
+} from '@/types/modules';
+
```

Comment:
```suggestion
defineOptions({
  inheritAttrs: false
})
```

Since we are wrapping the component and passing the attrs ourselves we should disable the attribute inheritance